src/includes/Helpers/CartHelper.php

```diff
@@ -184,7 +193,7 @@ public function get_cart_eligibilities() {
 	/**
 	 * Get eligible plans keys for current cart.
 	 *
-	 * @param array $cart_eligibilities The eligibilities.
+	 * @param array|Eligibility $cart_eligibilities The eligibilities.
```

Comment:
can't be an eligibility object - must be an Eligibility[]

Reply:
done